src/components/LayerCollection/LayerPOLine.js

```diff
@@ -131,25 +131,14 @@ class LayerPOLine extends Component {
     const line = cloneDeep(data);
     const { parentMutator, location: { pathname } } = this.props;
 
-    // remove sample data from payload to prevent errors on saving sample lines
```

Comment:
I've decided to remove this block since sample data now should be in a good shape.